streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/config/master/ResourceConfig.java

```diff
@@ -0,0 +1,45 @@
+package org.ray.streaming.runtime.config.master;
+
+import org.ray.streaming.runtime.config.Config;
+
+/**
+ * Job resource management config.
+ */
+public interface ResourceConfig extends Config {
+
+  String RESOURCE_KEY_CPU = "CPU";
+  String RESOURCE_KEY_MEM = "MEM";
+
+  String MAX_ACTOR_NUM_PER_CONTAINER = "streaming.container.per.max.actor";
+  String CONTAINER_RESOURCE_CHECk_INTERVAL_SECOND = "streaming.resource.check.interval.second";
+
+  String TASK_RESOURCE_CPU = "streaming.task.resource.cpu";
+  String TASK_RESOURCE_MEM = "streaming.task.resource.mem";
+  String TASK_RESOURCE_CPU_LIMIT_ENABLE = "streaming.task.resource.cpu.limitation.enable";
+  String TASK_RESOURCE_MEM_LIMIT_ENABLE = "streaming.task.resource.mem.limitation.enable";
+
+  @DefaultValue(value = "1.0")
+  @Key(value = TASK_RESOURCE_CPU)
+  double taskCpuResource();
```

Comment:
> Won't it be more reasonable for every operator to have it's own resources config such as `data_stream.map(...).resource(...)`, rather than config globally?

 This is used to giving a default value. The ```resource()``` api will override the resource config value here.

Reply:
Can you put this part code into resource PR?  It seems is not related to graph.